.github/workflows/stalebot.yml

```diff
@@ -0,0 +1,24 @@
+name: Close stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 9 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: "Hi, It's been a while since we last heard from you. We are marking this issue as stale due to inactivity. Please, provide the requested feedback, or we will have to close the issue in the next 7 days."
+        stale-pr-message: "Hi, It's been a while since we last heard from you. We are marking this pull request as stale due to inactivity. Please, provide the requested feedback, or we will have to close the pull request in the next 7 days."
+        close-issue-message: "This issue has not been active for 14 days already. We're closing it for now. Still, feel free to provide us requested feedback so that we can reopen it."
+        close-pr-message: "This pull request has not been active for 14 days already. We're closing it for now. Still, feel free to provide us requested feedback so that we can reopen it."
+        stale-issue-label: 'status:stale'
+        stale-pr-label: 'status:stale'
```

Comment:
I wonder if we should use the same labels for PRs. Generally makes sense, OTOH we never labeled PRs so it might be a little odd on the beginning. Maybe different labels would make it easier to distinguish what is the purpose here and there? Something simple like `pr:stale`?

Also for closed PRs we can add our `pr:frozen` label (with `close-pr-label:`) so it will make sense to have `pr:*` labels for better consistency. And for closed issues, we should add `resoltuion:expired` probably?

Reply:
Using the same label seems just simpler. Not sure why we would need to distinguish both, because they are still from different views (https://github.com/ckeditor/ckeditor4/issues vs https://github.com/ckeditor/ckeditor4/pulls). But since we already are doing such distinction, I have no problem changing it.

> And for closed issues, we should add resoltuion:expired probably?

I see there is an option for that, I will update PR accordingly.